src/rev_ai/customvocabulariesclient.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using Rev.ai"""
+
+import requests
+import json
+from requests.exceptions import HTTPError
+# from .models import Job, Account, Transcript, CaptionType
+from . import __version__
+
+try:
+    from urllib.parse import urljoin
+except:
+    from urlparse import urljoin
+
+
+class RevAiCustomVocabulariesClient:
+    """Client which implements Rev.ai CustomVocabulary API"""
+
+    # Default version of Rev.ai
+    version = 'v1'
+
+    # Default address of the API
+    base_url = 'https://api.rev.ai/speechtotext/{}/vocabularies'.format(
```

Comment:
@kbridbur , Do we want to drop Python 2.7 support? If we do, we can start to use python fstrings

Reply:
We should continue with Python2.7 support until the end of its lifetime (Jan 1st 2020)